src/harness/fourslash.ts

```diff
@@ -2348,7 +2352,7 @@ namespace FourSlash {
         private applyCodeActions(actions: ts.CodeAction[], index?: number): void {
             if (index === undefined) {
                 if (!(actions && actions.length === 1)) {
-                    this.raiseError(`Should find exactly one codefix, but ${actions ? actions.length : "none"} found.`);
+                    this.raiseError(`Should find exactly one codefix, but ${actions ? actions.length : "none"} found. ${actions ? actions.map(a => `"${a.description}"`).join(", ") : "" }`);

```

Comment:
Yeah, I had it that way at first, but it looks silly when ```actions.length``` is zero. Instead of adding fancy formatting logic I went with something that looks sort of okay in either case :)